setup.py

```diff
@@ -1,46 +1,47 @@
-import os
-from io import open
-from setuptools import setup, find_packages
+# Automatically generated from poetry/pyproject.toml
+# flake8: noqa
+# -*- coding: utf-8 -*-
+from setuptools import setup
 
+packages = \
+['c7n',
+ 'c7n.actions',
+ 'c7n.filters',
+ 'c7n.reports',
+ 'c7n.resources',
+ 'c7n.ufuncs']
 
-def read(fname):
-    return open(os.path.join(os.path.dirname(__file__), fname), encoding='utf-8').read()
+package_data = \
```

Comment:
this seems to pull in the whole source tree, if using python setup.py sdist, it works fine for poetry build which uses the package metadata specs that we have in pyproject.toml.

Reply:
it only seems to happen for c7n_azure, vs other packages. i've tried excludes with in MANIFEST.in  but it doesn't seem to be taking. for now i'm just going to roll with it as is, and file a bug to have a look at this issue separately.

Reply:
spoke too soon, it also happens with the main c7n package, pulling in tests and tools.